<launch>
    <arg name="ifcb_dataset" default="radbot" />
    <arg name="ifcb_dashboard_url" default="https://habon-ifcb.whoi.edu" />
    <arg name="topic" value="/ifcb" />
    <arg name="gps_fix_topic" default="/gps/fix" />
    <node pkg="fake_ifcb" type="node.py" name="fake_ifcb">
        <param name="~ifcb_dataset" value="$(arg ifcb_dataset)" />
        <param name="~ifcb_dashboard_url" value="$(arg ifcb_dashboard_url)" />
        <param name="~topic" value="$(arg topic)" />
        <param name="~gps_fix_topic" value="$(arg gps_fix_topic)" />
    </node>
</launch>